iocore/net/SSLConfig.cc

```diff
@@ -286,6 +290,30 @@ SSLConfigParams::initialize()
   ssl_client_ctx_options |= SSL_OP_NO_SESSION_RESUMPTION_ON_RENEGOTIATION;
 #endif
 
+#if TS_OPENSSL_1_1_1_API
+  REC_ReadConfigInteger(server_max_early_data, "proxy.config.ssl.server.max_early_data");
+  REC_ReadConfigInteger(server_recv_max_early_data, "proxy.config.ssl.server.recv_max_early_data");
+  Debug("ssl.early_data", "server_max_early_data = %d, server_recv_max_early_data = %d", server_max_early_data,
+        server_recv_max_early_data);
+
+  if (server_max_early_data < 0) {
+    Debug("ssl.early_data", "proxy.config.ssl.server.max_early_data = %d: Must be positive.", server_max_early_data);
+    server_max_early_data = 0;
+  }
+
+  if (server_recv_max_early_data < 0) {
+    Debug("ssl.early_data", "proxy.config.ssl.server.recv_max_early_data = %d: Must be positive.", server_recv_max_early_data);
```

Comment:
Don't Debug() error cases, error them properly so the user knows to fix the configs.

Reply:
Done.